doc/10-icinga-template-library.md

```diff
@@ -525,6 +534,7 @@ Custom variables passed as [command parameters](03-monitoring-basics.md#command-
 
 Name               | Description
 -------------------|-------------------
+game_extra_opts    | **Optional.** Read extra plugin options from an ini file.
```

Comment:
Also put this after `game_ipaddress` (keeping all the required variables above it).